dashboard/public/pixelation/pixelation.html

```diff
@@ -90,8 +90,42 @@
     $('#below_viz_instructions').text(options.instructions).show();
   }
   pixelationInit();
-  pixel_data.value = options.data;
-  drawGraph();
-  if(options.format=='raw') unformatBits()
-  else formatBitDisplay();
+
+  function pixelationDisplay() {
+    pixel_data.value = options.data;
+    drawGraph();
+    formatBitDisplay();
+  }
+
+  // Initialize projects for save/load functionality if channel id is present.
+  if (appOptions.channel) {
+    window.apps.setupProjectsExternal();
+    var projectsHandler = {
+      setInitialLevelHtml: function (levelHtml) {},
+      getLevelHtml: function () {
+        return '';
+      },
+      setInitialLevelSource: function (levelSource) {
+        options.data = levelSource;
```

Comment:
The "contract" for appOptions is fuzzy to me. Ideally, would appOptions modified only by dashboard (and, conversely, never modified by the app?)